packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv6/ipv6.controller.js

```diff
@@ -0,0 +1,164 @@
+        this.ipv6Catalog = this.getIpv6Catalog();
+        if (this.$state.params.service) {
+          this.model.selectedService = find(this.services, {
+            serviceName: this.$state.params.service.serviceName,
+          });
+        }
+      })
+      .catch((err) => {
+        this.Alerter.error(
+          this.$translate.instant('ip_order_loading_error', this.ALERT_ID),
+        );
+        return this.$state
+          .go(DASHBOARD_STATE_NAME)
+          .then(() => this.$q.reject(err));
+      })
+      .finally(() => {
+        this.loading.services = false;
+      });
+  }
+
+  getIpv6Catalog() {
+    return this.ipCatalog.filter(
+      (plan) => plan.planCode.match(/^ip-v6.*/) != null,
+    );
+  }
+
+  loadRegions() {
+    this.catalogByLocation = this.ipv6Catalog.map((plan) => {
+      const {
+        details: {
+          product: { configurations },
+        },
+      } = plan;
+      const regionConfig = configurations.find(
+        (config) => config.name === 'ip_region',
+      );
+      const regionId = regionConfig.values[0] || '';
+      const countryCode = this.constructor.getMacroRegion(regionId);
+
+      return {
+        regionId,
+        planCode: plan.planCode,
+        location: this.$translate.instant(`ip_agora_ipv6_location_${regionId}`),
+        icon: `oui-flag oui-flag_${FLAGS[countryCode]}`,
+      };
+    });
+  }
+
+  static getMacroRegion(region) {
+    const localZonePattern = /^lz/i;
+    const devZonePattern = /^1-/i;
+
+    let macro;
+    const local = region
+      .split('-')
+      ?.slice(2)
+      ?.join('-');
+
+    if (devZonePattern.test(local)) {
+      macro = [local];
+    } else {
+      const nbOfSlice = localZonePattern.test(local) ? 3 : 2;
+      macro = /[\D]{2,3}/.exec(
+        region
+          .split('-')
+          ?.slice(nbOfSlice)
+          ?.join('-'),
+      );
+    }
+
+    return (macro && macro[0]) || '';
+  }
+
+  redirectToPaymentPage() {
+    const { planCode, regionId } = this.model.selectedPlan;
+    const productToOrder = this.IpAgoraOrder.constructor.createProductToOrder({
+      productId: 'ip',
+      pricingMode: 'default',
+      regionId,
+      duration: 'P1M',
+      planCode,
+      quantity: 1,
+    });
+
+    if (this.model.selectedService !== EMPTY_CHOICE) {
+      productToOrder.destination = this.model.selectedService;
+    }
+
+    return this.User.getUrlOf('express_order')
+      .then((url) => {
+        this.$window.open(
+          `${url}review?products=${JSURL.stringify([productToOrder])}`,
+          '_blank',
+        );
+        this.$state.go(DASHBOARD_STATE_NAME);
+      })
+      .catch((err) => {
+        this.Alerter.error(
+          this.$translate.instant('ip_order_finish_error'),
+          this.ALERT_ID,
+        );
+        return this.$state
+          .go(DASHBOARD_STATE_NAME)
+          .then(() => this.$q.reject(err));
+      });
+  }
+
+  resumeOrder() {
+    return this.$state.go(DASHBOARD_STATE_NAME);
+  }
```

Comment:
The name is confusing. 
Moreover, you can replace `this.$state.go(DASHBOARD_STATE_NAME)` in the controller with that function
```suggestion
  goToDashboard() {
    return this.$state.go(DASHBOARD_STATE_NAME);
  }
```